Competition/src/main/include/Muncher.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "ValorSubsystem.h"
+#include "Constants.h"
+
+#include <frc/XboxController.h>
+#include <frc/PWMVictorSPX.h>
+
+#ifndef MUNCHER_H
+#define MUNCHER_H
+
+class Muncher : public ValorSubsystem {
+    public:
+        Muncher();
+        void setController(frc::XboxController*);
+
+        void initMuncher();
```

Comment:
You'll have to change this to match the base class.

I just merged @rohitchawla28's changes to the base class and it's on `dev`. The rebase won't go smoothly without this change